thamos/lib.py

```diff
@@ -1286,6 +1287,32 @@ def get_verified_packages_from_static_analysis(
         try:
             imported_packages = get_package_from_imported_packages(import_name)
         except ApiException as exc:
+            error_message = "({0})\n Reason: {1}\n".format(exc.status, exc.reason)
```

Comment:
Would it make sense to raise a thamos specific exception with a message like:

```
"Failed to obtain package for import {import_name!r} (HTTP status %r): {body['error']}"
```

We already have exception handling in thamos that behaves based on verbosity (ex. `thamos --verbose whatprovides nonexisting` and `thamos whatprovides nonexisting` - the first one reports back full stacktrace). The logic might parse the response more accuratelly (as it does not need to be always JSON from the server in very specific cases).

Reply:
You're right, all the details in the response body are already available in `--verbose` mode so let's keep the essential message :+1: